cumulusci/tasks/salesforce/ListCommunityTemplates.py

```diff
@@ -0,0 +1,26 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ListCommunityTemplates(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Lists Salesforce Community templates available for the current org via the Connect API.
+    """
+    task_options = {}
+
+    def _init_options(self, kwargs):
+        super(ListCommunityTemplates, self)._init_options(kwargs)
+
+    def _run_task(self):
+        community_template_list = self.sf.restful("connect/communities/templates")[
+            "templates"
+        ]
+
+        community_template_names = {
+            t["templateName"]: t for t in community_template_list
+        }
+
+        self.logger.info(
+            "Community Templates available to the current org:%s",
+            "\r\n".join(community_template_names),
```

Comment:
Why not just do

    "\n".join(t["templateName"] for t in community_template_list)

and skip defining `community_template_names`

Reply:
Flake8 rejects that as invalid syntax. I had attempted it, though.

Reply:
It doesn't for me -- what was the error from flake8?

Reply:
It's working now. It must have been a typo on my part.